src/main/java/com/datapipe/jenkins/vault/credentials/VaultLdapUsingChefServerCredential.java

```diff
@@ -0,0 +1,145 @@
+package com.datapipe.jenkins.vault.credentials;
+
+import com.bettercloud.vault.Vault;
+import com.bettercloud.vault.VaultConfig;
+import com.bettercloud.vault.VaultException;
+import com.cloudbees.plugins.credentials.CredentialsScope;
+import com.cloudbees.plugins.credentials.impl.BaseStandardCredentials;
+import com.datapipe.jenkins.vault.exception.VaultPluginException;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import hudson.Extension;
+import org.jclouds.ContextBuilder;
+import org.jclouds.chef.ChefApi;
+import org.jclouds.chef.ChefApiMetadata;
+import org.jclouds.chef.domain.DatabagItem;
+import org.kohsuke.stapler.DataBoundConstructor;
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+
+public class VaultLdapUsingChefServerCredential extends BaseStandardCredentials implements VaultCredential {
+
+    private final @Nonnull
+    String serviceAccountName;
```

Comment:
ditto on annotation being on a separate line and field name being on the same line as private